awesome-tray.el

```diff
@@ -1200,10 +1200,12 @@ If right is non nil, replace to the right"
   "Get new text to be displayed."
   (interactive)
   (let* ((tray-info (awesome-tray-build-active-info))
-         (minibuffer-info (current-message))
+         (minibuffer-info (or (current-message) ""))
+         (minibuffer-info (set-text-properties 0 (length minibuffer-info) nil minibuffer-info))
+         (minibuffer-info (if (stringp minibuffer-info) minibuffer-info ""))
```

Comment:
After `(minibuffer-info (or (current-message) ""))`, minibuffer-info must be string, why add this?

Reply:
When there is no information displayed in the minibuffer, `(current-message)` will return nil. So it is necessary to ensure that minibuffer-info is a string, otherwise the operations on the string below will result in problems.